nearlib/test/integration.test.js

```diff
@@ -41,6 +41,27 @@ describe('dev connect', () => {
         };
     });
 
+    test('test dev connect like template', async () => {
+        window.localStorage = createFakeStorage();
+        // Mocking some
+        let tmpCreate = dev.createAccountWithContractHelper;
+        let devConfig = dev.getConfig;
+        dev.getConfig = async () => "THE_CONFIG";
+        dev.createAccountWithContractHelper = async (nearConfig, newAccountId, publicKey) => {
+            expect(nearConfig).toEqual("THE_CONFIG");
+            return await dev.createAccountWithLocalNodeConnection(newAccountId, publicKey);
+        };
+        // Calling
+        let near = await dev.connect();
+        // Restoring mocked functions
+        dev.getConfig = devConfig;
```

Comment:
Are we running all the tests serially now? This might cause flakiness if we are ever doing stuff in parallel.

Reply:
I think we run them sequentially right now (within a single file). promise tests are running in parallel, but they use different JS instance